Dockerfile

```
@@ -0,0 +1,13 @@
+FROM python:3.7
+COPY . /workdir
+WORKDIR /workdir
+RUN pip install --upgrade pip && pip install \
+    black \
+    codecov \
+    flake8 \
+    mutmut \
+    pandas \
+    pylint \
+    pytest-cov \
+    pytest==5.0.1 
```

Teníamos "amarrada" la versión 5.0.1 de Pytest porque había un _bug_ en una versión posterior 5.X.0, X>0. Recientemente, (julio 31) lanzaron la versión 6.0.1. Tal vez deberíamos probar esa nueva versión?

```suggestion
    pytest
```